### PR TITLE
Webpack

### PR DESCRIPTION
**Pull request by alexeagle**
Builds on #576 by adding an experimental `webpack_bundle` rule.
It doesn't have e2e testing yet, should add this similar to rollup.

### CODE REVIEW

**Comment by alexeagle** [comment]
Handing this PR to @gregmagolan to merge

**Comment by teddyknox** [comment]
Bump

**Comment by v48** [comment]
Is there any progress on this? I would like to use this in my project.

File: packages/labs/package.json [hunk 1]
```diff
@@ -11,9 +11,24 @@
     "bugs": {
         "url": "https://github.com/bazelbuild/rules_nodejs/issues"
     },
-    "devDependencies": {},
-    "dependencies": {},
+    "devDependencies": {
+        "@bazel/typescript": "^0.25.1",
```
Comment (gregmagolan) [inline packages/labs/package.json]: Should probably be `"@bazel/typescript": "bazel://@npm_bazel_typescript//:npm_package",` like /internal/karma/package.json

Comment (alexeagle) [reply packages/labs/package.json]: done

**Pull request merged**